crates/orderbook-commons/src/lib.rs

```diff
@@ -352,8 +354,38 @@ pub fn get_filled_with_from_matches(matches: Vec<Matches>) -> Result<FilledWith>
     })
 }
 
+/// Calculates the expiry timestamp at the next Sunday at 3 pm UTC from a given offset date time.
+/// If the argument falls in between Friday, 3 pm UTC and Sunday, 3pm UTC, the expiry will be
+/// calculated to next weeks Sunday at 3 pm
+pub fn get_expiry_timestamp(time: OffsetDateTime) -> OffsetDateTime {
```

Comment:
:wrench: I think we should rename this to `calculate_next_expiry` or something like that.